tests/unit/TaskManagement/Controller/SharesControllerTest.php

```diff
@@ -12,6 +12,7 @@
 use Ora\StreamManagement\Stream;
 use Rhumsaa\Uuid\Uuid;
 use Ora\User\User;
+use Ora\ReadModel\Organization as ReadModelOrganization;
```

Comment:
Immagino allora che contestualmente sia da cambiare anche il costruttore di Ora\StreamManagement\Stream che prende in ingresso un variabile $organization della classe Ora\ReadModel\Organization